medical_ui/catalog/services.py

```
@@ -0,0 +1,120 @@
+    job_id: str
+    status: str = "queued"
+    manifest: list[dict[str, Any]] = field(default_factory=list)
+    error: str | None = None
+
+
+JOBS: dict[str, DiscoveryJob] = {}
+JOBS_LOCK = Lock()
+EXECUTOR = ThreadPoolExecutor(max_workers=2, thread_name_prefix="medical-discovery")
+
+
+def _load_medical_script() -> Any:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_load_medical_script`

(ANN401)